app/helpers/settings_helper.rb

```diff
@@ -25,6 +25,10 @@ def short_cat(category)
     category.gsub(/Setting::/,'')
   end
 
+  def cat_label(category)
+    category.constantize.humanized_category || short_cat(category)
```

Comment:
`humanized_category` calls the deprecated Setting method_missing getter by default. Is there meant to be a default implementation in Setting?